weave/context_state.py

```diff
@@ -1,6 +1,7 @@
 import contextlib
```

Comment:
Should we note in the header that this is a "legacy" file? `WandbApiContext` is a query service concept

Reply:
yeah, it's moved in the next pr:
https://github.com/wandb/weave/pull/2471